package.json

```diff
@@ -13,24 +13,9 @@
     "test": "vitest",
     "eslint-check": "eslint src/",
     "lint-check": "prettier --config ./.prettierrc --ignore-path ./.prettierignore --check '**/*.{ts,tsx,js,jsx,json,html,css,yaml,md}'",
-    "lint-fix": "prettier --config ./.prettierrc --ignore-path ./.prettierignore --write '**/*.{ts,tsx,js,jsx,json,html,css,yaml,md}'"
```

Comment:
Are you going to adjust busola to the same order?
It's one of the acceptance criteria in technical improvements.